xontrib/vox.py

```diff
@@ -1,77 +1,99 @@
 """Python virtual environment manager for xonsh."""
 
 import sys as _sys
+import argparse as _argparse
 import xontrib.voxapi as _voxapi
 
 class _VoxHandler:
     """Vox is a virtual environment manager for xonsh."""
+    parser = _argparse.ArgumentParser(prog='vox', description=__doc__)
+    subparsers = parser.add_subparsers(dest='command')
+
+    create = subparsers.add_parser('new', help='Create a new virtual environment')
+    create.add_argument('name', metavar='ENV',
+                        help='The environments to create')
+
+    create.add_argument('--system-site-packages', default=False,
+                        action='store_true', dest='system_site',
+                        help='Give the virtual environment access to the '
+                             'system site-packages dir.')
+    from xonsh.platform import ON_WINDOWS
+    if ON_WINDOWS:
+        use_symlinks = False
+    else:
+        use_symlinks = True
```

Comment:
also you could set use symlinks to not on_windows, rather than going through an if statement